data/tlg0057/tlg075/tlg0057.tlg075.1st1K-grc1.xml

```diff
@@ -74,7 +74,8 @@
                             <date>1826</date>
                         </imprint>
                     </monogr>
-                    <ref target="https://catalog.hathitrust.org/Record/009723957">HathiTrust</ref>
+                    <ref target="https://hdl.handle.net/2027/hvd.hxpp8s?urlappend=%3Bseq=391%3Bownerid=27021597764275152-395">HathiTrust</ref><!-- Volume 11 -->
```

Comment:
I use an n attribute to add volume rather than a comment
such as n="vol_11"

Reply:
Gotcha, I will go back and change that in a final pass. Just one last Galen pull request to send your way after this!